layouts/millennium_dstu2_sidebar.html

```diff
@@ -0,0 +1,98 @@
+          <li><a href="/millennium/dstu2/general-clinical/procedure/">Procedure</a></li>
+        </ul>
+      </li>
+      <li class="js-topic">
+        <h3><a href="#" class="js-expand-btn collapsed arrow-btn" data-proofer-ignore></a><a href="/millennium/dstu2/diagnostic/">Diagnostic</a>
+        </h3>
+        <ul class="js-guides">
+          <li><a href="/millennium/dstu2/diagnostic/diagnostic-report/">DiagnosticReport</a></li>
+          <li><a href="/millennium/dstu2/diagnostic/observation/">Observation</a></li>
+        </ul>
+      </li>
+      <li class="js-topic">
+        <h3><a href="#" class="js-expand-btn collapsed arrow-btn" data-proofer-ignore></a><a href="/millennium/dstu2/devices/">Devices</a>
+        </h3>
+        <ul class="js-guides">
+          <li><a href="/millennium/dstu2/devices/device/">Device</a></li>
+        </ul>
+      </li>
+      <li class="js-topic">
+        <h3><a href="#" class="js-expand-btn collapsed arrow-btn" data-proofer-ignore></a><a href="/millennium/dstu2/encounter-workflow/">Encounter Workflow</a>
+        </h3>
+        <ul class="js-guides">
+          <li><a href="/millennium/dstu2/encounter-workflow/encounter/">Encounter</a></li>
+        </ul>
+      </li>
+      <li class="js-topic">
+        <h3><a href="#" class="js-expand-btn collapsed arrow-btn" data-proofer-ignore></a><a href="/millennium/dstu2/entities/">Entities</a>
+        </h3>
+        <ul class="js-guides">
+          <li><a href="/millennium/dstu2/entities/contract/">Contract</a></li>
+          <li><a href="/millennium/dstu2/entities/person/">Person</a></li>
+        </ul>
+      </li>
+      <li class="js-topic">
+        <h3><a href="#" class="js-expand-btn collapsed arrow-btn" data-proofer-ignore></a><a href="/millennium/dstu2/infrastructure/">Infrastructure</a>
+        </h3>
+        <ul class="js-guides">
+          <li><a href="/millennium/dstu2/infrastructure/binary/">Binary</a></li>
+          <li><a href="/millennium/dstu2/infrastructure/document-reference/">DocumentReference</a></li>
+        </ul>
+      </li>
+      <li class="js-topic">
+        <h3><a href="#" class="js-expand-btn collapsed arrow-btn" data-proofer-ignore></a><a href="/millennium/dstu2/individuals/">Individuals</a>
+        </h3>
+        <ul class="js-guides">
+          <li><a href="/millennium/dstu2/individuals/patient/">Patient</a></li>
+          <li><a href="/millennium/dstu2/individuals/related-person/">RelatedPerson</a></li>
+        </ul>
+      </li>
+      <li class="js-topic">
+        <h3><a href="#" class="js-expand-btn collapsed arrow-btn" data-proofer-ignore></a><a href="/millennium/dstu2/medications/">Medications</a>
+        </h3>
+        <ul class="js-guides">
+          <li><a href="/millennium/dstu2/medications/immunization/">Immunization</a></li>
+          <li><a href="/millennium/dstu2/medications/medication-statement/">MedicationStatement</a></li>
```

Comment:
nit: since you were the one to bring up alphabetical order, swap this with medication order. :)